### ISSUE TITLE
ssh excluteable and connection refused 

### ISSUE THREAD

**Issue by ikennarene** [dtag-dev-sec/tpotce#26]
hi, i cant find (ssh_enable.sh) in the same session i found (2fa_enable.sh). and when i try connect to ssh based on the installation given i get port 64295: connection refused. pls can i get a deep details on how to do it. Thanks

**Comment by dev-t-sec** [comment]
Dear Ikennarene, Per default ssh is disabled, the ssh enable script is located
in the same directory as the 2fa script, only after running it, ssh is possible. How many memory and diskspace you use ? Best wishes
Markus

On 19 Mar 2016, at 09:27, ikennarene <notifications@github.com<mailto:notifications@github.com>> wrote:


hi, i cant find (ssh_enable.sh) in the same session i found (2fa_enable.sh). and when i try connect to ssh based on the installation given i get port 64295: connection refused. pls can i get a deep details on how to do it. Thanks

**Comment by t3chn0m4g3** [comment]
With T-Pot 16.03 there is no ssh_enable script. Please see this [video ](https://youtu.be/6GmIxVJ6eAQ) how to proceed.

**Comment by ikennarene** [comment]
memory 4gb, diskspace 83gb.  Now when i run ssh -p 64295 -l tsec -N -L8080:127.0.0.1:64296 192.168.x.x, i get ( ssh: connect to host 192.168.x.x port 64295: connection refused

**Comment by t3chn0m4g3** [comment]
From the error message I assume SSH is still not enabled. Please check with `status ssh`

**Comment by ikennarene** [comment]
showed ssh start/running

**Comment by t3chn0m4g3** [comment]
did you enable two-factor or are you using a different authentication type (public key, password, etc...)?

**Comment by ikennarene** [comment]
i have not done any of them.

**Comment by t3chn0m4g3** [comment]
That is why you get a connection refused `pam.d` does not know how to authenticate you. You need to enable two-factor-authentication, but you then have to reboot.

**Comment by ikennarene** [comment]
okay, Thanks. let me do that

**Comment by t3chn0m4g3** [comment]
Welcome, you can always follow the installation step by step in the [tutorials](https://www.youtube.com/playlist?list=PLBRkfSbyxi-ArIfQv-3XOaKun1UpZnr0X).

**Comment by ikennarene** [comment]
hi all, now i can connect , but i can not access the kibana dashboard. (This site can’t be reached

127.0.0.1 refused to connect.)

**Comment by schmalle** [comment]
Hi, Did you establish the ssh tunnel ? (64296 is only available from localhost, so you must create a port forwardning) Best wishes Markus

**Comment by ikennarene** [comment]
@schmalle  Pls, you mean this ssh -p 64295 -l tsec -L8080:127.0.0.1:64296 192.168.153.143   , if yes, already done this and okay

**Comment by ikennarene** [comment]
hi, i have done the installation, and verification code. I have installed putty and OpenSSH. Base on the video https://www.youtube.com/watch?v=A3LF63os17k , when open the OpenSSH folder, there should be a key, but i cant find in my own, i can only see known_host. Please is there anything am missing

**Comment by t3chn0m4g3** [comment]
t-pot itself works without ssh keys (see the end of the video) and uses two-factor-authentication. if you want to use key based authentication follow this [guide](https://help.ubuntu.com/community/SSH/OpenSSH/Keys) and the appropriate t-pot [info](http://dtag-dev-sec.github.io/mediator/feature/2016/03/11/t-pot-16.03.html#ssh).

**Comment by jjjan** [comment]
hello according to video when i use 127.0.0.1:8080 is not show me kibana page?
so am i should change proxy section of browser? could you explain more

**Comment by dev-t-sec** [comment]
Dear Mark,

Pls have a look at the document itself, you must tunnel to you tpot installation. (Kibana dashboard secrin)
In the video it is expected that you forward the local port 8080 to the remote system.

Best regards

Markus

On 23 Mar 2016, at 10:56, mark parkour <notifications@github.com<mailto:notifications@github.com>> wrote:


hello according to video when i use 127.0.0.1:8080 is not show me kibana page?
so am i should change proxy section of browser? could you explain more
